hardware/products/ucm/intro.md

```diff
@@ -1,3 +1,249 @@
+| 93			| SPI B MISO					|
+| 94			| SPI B MOSI					|
+| 95			| GND							|
+| 96			| SPI B SCK						|
+| 97			| ADC B							|
+| 98			| CAN A TD						|
+| 99			| CAN A RD						|
+| 100			| CAN B TD						|
+| 101			| CAN B RD						|
+| 102			| UART HS A TX					|
+| 103			| UART HS A RX					|
+| 104			| ADC C							|
+| 105			| PWM A							|
+| 106			| 3.3V							|
+| 107			| Module Specific 1				|
+| 108			| Module Specific 2				|
+| 109			| Module Specific 3				|
+| 110			| ADC D							|
+| 111			| Bootstrap C					|
+| 112			| PWM B							|
+| 113			| GND							|
+| 114			| ADC E							|
+| 115			| I2C A SDA						|
+| 116			| I2C A SCL						|
+| 117			| UART A RX						|
+| 118			| UART A TX						|
+| 119			| GPIO I						|
+| 120			| UART HS A RTS					|
+| 121			| UART HS A CTS					|
+| 122			| GPIO J						|
+| 123			| SD Card D0					|
+| 124			| 3.3V							|
+| 125			| SD Card CMD					|
+| 126			| SD Card CLK					|
+| 127			| SD Card D1					|
+| 128			| SD Card D2					|
+| 129			| SD Card D3					|
+| 130			| PWM C							|
+| 131			| GND							|
+| 132			| GPIO K						|
+| 133			| PWM D							|
+| 134			| Bootstrap B					|
+| 135			| Bootstrap D					|
+| 136			| GPIO L						|
+| 137			| Module Specific 10			|
+| 138			| UART HS B RTS					|
+| 139			| UART HS B CTS					|
+| 140			| UART HS B TX					|
+| 141			| UART HS B RX					|
+| 142			| 3.3V							|
+| 143			| LCD VSYNC						|
+| 144			| LCD HSYNC						|
+| 145			| LCD CLK						|
+| 146			| LCD DE						|
+| 147			| Module Specific 11			|
+| 148			| SD Card CD					|
+| 149			| Module Specific 12			|
+| 150			| Reserved						|
+| 151			| GND							|
+| 152			| LCD B3						|
+| 153			| LCD B4						|
+| 154			| LCD B5						|
+| 155			| LCD B6						|
+| 156			| LCD B7						|
+| 157			| ADC F							|
+| 158			| ADC G							|
+| 159			| ADC H							|
+| 160			| 3.3V							|
+| 161			| LCD G2						|
+| 162			| LCD G3						|
+| 163			| LCD G4						|
+| 164			| LCD G5						|
+| 165			| LCD G6						|
+| 166			| Module Specific 13			|
+| 167			| Reserved						|
+| 168			| LCD R7						|
+| 169			| GND							|
+| 170			| LCD G7						|
+| 171			| LCD R3						|
+| 172			| LCD R4						|
+| 173			| LCD R5						|
+| 174			| LCD R6						|
+| 175			| SPI A SCK						|
+| 176			| SPI A MISO					|
+| 177			| Module Specific 14			|
+| 178			| SPI A MOSI					|
+| 179			| Module Specific 15			|
+| 180			| 3.3V							|
+| 181			| Module Specific 16			|
+| 182			| Module Specific 17			|
+| 183			| VBAT							|
+| 184			| Module Specific 18			|
+| 185			| GND							|
+| 186			| GND							|
+| 187			| RESET							|
+| 188			| USB Host D+					|
+| 189			| JTAG RTCK						|
+| 190			| USB Host D-					|
+| 191			| JTAG TDO						|
+| 192			| 3.3V							|
+| 193			| JTAG NTRST					|
+| 194			| USB Device D+					|
+| 195			| JTAG TDI						|
+| 196			| USB Device D-					|
+| 197			| JTAG TCK (SWCLK)				|
+| 198			| GND							|
+| 199			| JTAG TMS (SWDIO)				|
+| 200			| Reserved						|
+
+## UCM Standard - Module Dimesions
```

Comment:
Do a mockup in EAGLE or Altium for the dimensions instead of these images. It took me a minute to figure out what exactly we were trying to say here. Add a short description too.

Reply:
This cannot be done until I/we discuss this with Gus. I am not sure of the sizes or if Gus wants a specific size defined anymore. The sizes that I chose were the original and the extra large one and the middle was just the mid size in between the other two.

Reply:
Alright